src/app/services/project.service.ts

```diff
@@ -92,15 +92,22 @@ export class ProjectService {
   }
 
   private addCollection(collectionsList: CollectionsList, collection: Collection) {
+    // ensure the collection hasn't already been added before adding again.
     switch (collection.parentType) {
       case 'Authorizations':
-        collectionsList.authorizations[collection.agency].add(collection);
+        if (!collectionsList.authorizations[collection.agency].items.some(existing => existing._id === collection._id)) {
```

Comment:
Do we want any null checks on any of these? Not sure if these properties are guaranteed or not.

Reply:
Yeah couldn't hurt. I'm almost hesitant to even bother with the fix as we're going to be re-writing how collections work in the next sprint anyway

Reply:
Very true